package.json

```diff
@@ -24,12 +24,18 @@
   "homepage": "https://github.com/imbcmdth/hls-fetcher#readme",
   "dependencies": {
     "aes-decrypter": "^1.0.2",
+    "async": "^2.6.1",
     "bluebird": "^3.4.0",
     "m3u8-parser": "^1.0.2",
     "mkdirp": "^0.5.1",
     "pessimist": "^0.3.5",
     "promise-streams": "^1.0.1",
+    "request": "^2.87.0",
```

Comment:
should this also have a browser compatible equivalent? (not sure how that stuff fits together)

Reply:
are we trying to make this module work in the browser as well as in node?

Reply:
doesn't it currently? We don't care but I was just concerned about regression.

Reply:
I don't think the intent was ever for this to run in the browser. only in node.

Reply:
cool, n/m then 🙈

Reply:
Right, the project's not set up to work in the browser